### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bogenschlag**
Some rewrapping/clean-up work.

### CODE REVIEW

**Comment by klees** [comment]
Hi @alex40724,

your my best guess for "correct assignee". I hope that's ok with you.

Best regards!

**Comment by alex40724** [comment]
Thanks, looks good to me. If nobody objects, I will merge this in the next days.

File: Customizing/README.md [hunk 1]
```diff
@@ -4,70 +4,90 @@
...
 1. [Skins and Styles](#skins-and-styles)
+1. [Plugins](#plugins)
 
 <!-- /MarkdownTOC -->
 
 <a name="introduction"></a>
-# Introduction
+## Introduction
 
 This directory holds all customized files for this ilias installation.
```
Comment (matthiaskunkel) [inline Customizing/README.md]: It would be great if we always write ILIAS in capital letters, thanks!

Comment (bogenschlag) [reply Customizing/README.md]: You are of course right. I had not expected such an "error" and simply overlooked it. I'll amend the pull request.

**Comment by Amstutz** [comment]
@bogenschlag : Yeah.

@alex40724 : Thx. If possible, also promote to trunk.

**Pull request merged**